README.markdown

```diff
@@ -178,7 +178,7 @@ autocmd BufEnter * if bufname('#') =~ 'NERD_tree_\d\+' && bufname('%') !~ 'NERD_
 
 ```vim
 " Open the existing NERDTree on each new tab.
-autocmd BufWinEnter * if getcmdwintype() == '' | silent NERDTreeMirror | endif
+autocmd BufWinEnter * if &buftype != 'quickfix' && getcmdwintype() == '' | silent NERDTreeMirror | endif
```

Comment:
This is already a known solution to this problem so I can't see why it shouldn't be merged into the master.
@alerque Is there some reason for holding this PR back?

Reply:
Nope, just that it was posted before I was monitoring contributions and haven't gone back and done issue triage. That's exactly the kind of thing I've been trying to convince people to contribute towards in #1280.

Reply:
I look forward to your approval on this PR so we can finally merge it into the master branch